frappe/public/js/frappe/ui/filters/filter_list.js

```diff
@@ -132,7 +133,7 @@ frappe.ui.FilterGroup = class {
 
 	get_filters() {
 		return this.filters.filter(f => f.field).map(f => {
-			f.freeze();
+			// f.freeze();
```

Comment:
Not sure if this is required. Removing this should fix the issue where ID=undefined gets set automatically too.

Reply:
Let's remove this and see! Even I don't understand why it exists here